packages/storybook/stories/Components/FileUploaderField.stories.tsx

```diff
@@ -21,10 +22,36 @@ export const fileUploaderField = createControlledStory(undefined, {
     uploadAgainMessage: "Upload another file: ",
     uploadingMessage: "Uploading...",
     uploadButtonLabel: "Choose a file",
-    assistiveTextFileTypes: (fileTypes) =>
+    assistiveTextFileTypes: (fileTypes?: Record<string, string[]>) =>
       fileTypes ? "Allowed file types: " + Object.values(fileTypes).flat().join(", ") : "",
-    assistiveTextMaxSize: (maxSizeMb) => (maxSizeMb ? "Max file size: " + maxSizeMb + "MB" : ""),
+    assistiveTextMaxSize: (maxSizeMb?: number) =>
+      maxSizeMb ? "Max file size: " + maxSizeMb + "MB" : "",
   },
   renderIssue: () => "error",
   maxFileSize: 1000,
+};
+
+export const fileUploaderField = createControlledStory(undefined, fileUploaderProps);
+
+export const loading = createControlledStory(undefined, {
+  ...fileUploaderProps,
+  isUploading: true,
 });
+loading.parameters = {
+  chromatic: { pauseAnimationAtEnd: true },
+};
```

Comment:
I'm not sure if this is the right parameter to ensure consistency between screenshots, I've also seen used `delay` in `AreaLoader`'s stories.